partials/shop-invoicetotals.htm

```diff
@@ -5,22 +5,32 @@
     {% set cartTotalNoTax = cartTotalNoTax + item.price %}
   {% endfor %}
 {% endfor %}
-
+{% set discount = totals.discount_total %}
 {% if cartTotalNoTax > 0 %}
-    <li class="list-group-item"><strong>Cart Total:</strong> <i>{{ cartTotalNoTax|currency }}</i></li>
+    <li class="list-group-item">Item Total: <div class="pull-right">{{ cartTotalNoTax|currency }}</div></li>
```

Comment:
If I make the change it will cause a merge conflict on the next branch, so I'll take a note of it and do another branch at the end to clean up all the minor fixes you bring up.